lib/AcqUnits/AcqUnits.js

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Field,
+} from 'redux-form';
+import { get, uniq, find } from 'lodash';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+
+import { MultiSelection } from '@folio/stripes/components';
+
+import { LIMIT_MAX } from '../constants';
+import { usePrevious } from '../utils';
+
+const itemToString = item => item;
+const label = <FormattedMessage id="stripes-acq-components.label.acqUnits" />;
+
+const AcqUnits = ({ name, resources, mutator, stripes }) => {
+  const userId = get(stripes, 'user.user.id');
+  const prevUserId = usePrevious(userId);
+
+  useEffect(() => {
+    if (userId !== prevUserId) {
```

Comment:
maybe we can avoid such logic? `stripes-connect` supports `params` attribute as an function, where props goes in. I think it might take less code in more declarative way